### DISCUSSION TITLE
Trame VR Support

### DISCUSSION THREAD

**Discussion by yangx456** [Kitware/trame#640] [General]
Hello, I was wondering if Trame app supports VR. Can we deploy the app on a VR headset for example Quest 3? Or can we access Trame web app through VR browser and visualize it like WebXR? Thank you!

**Comment by jourdain** [comment]
While trame can drive a VTK scene and probably push it to a VR environment, that is another story for the web side of things. We could see some WebXR component also integrate into trame, but we don't have any right now. I guess there are ways to make that happen to some extent, but right now we don't have anything specific to play with.
